Optimize file-type content modules (CQA) (#5056)
theforeman/foreman-documentation: guides/common/modules/proc_downloading-files-to-a-host-from-a-custom-file-repository-by-using-web-ui.adoc

= Downloading files to a host from a {customfiletype} repository by using {ProjectWebUI}

[role="_abstract"]
You can download files to a host over HTTPS and optionally over HTTP if the *Unprotected* option for the repository is selected.
Use {ProjectWebUI} to find the download URL of the {customfiletype} repository.

.Prerequisites
* You have a {customfiletype} repository.
ifdef::katello,satellite,almalinux,oracle_linux,red_hat_enterprise_linux,rocky_linux[]
For more information, see {ContentManagementDocURL}creating-a-custom-file-type-repository-by-using-web-ui[Creating a {customfiletype} repository by using {ProjectWebUI}] in _{ContentManagementDocTitle}_.
endif::[]
* You know the name of the file you want to download to hosts from the {customfiletype} repository.

> * To download the file over HTTPS, you require the following certificates on your host:

Reviewer: I don't think "you require" reads well, or is appropriate for prereqs.

```suggestion
* To download the file over HTTPS, your host has the following certificates:
```
